graph/loaders/comments.js

```diff
@@ -82,54 +82,52 @@ const getParentCountsByAssetID = (context, asset_ids) => {
 
 /**
  * Retrieves the count of comments based on the passed in query.
- * @param  {Object} context   graph context
+ * @param  {Object} ctx   graph context
  * @param  {Object} query     query to execute against the comments collection
  *                            to compute the counts
  * @return {Promise}          resolves to the counts of the comments from the
  *                            query
  */
-const getCommentCountByQuery = (context, {ids, statuses, asset_id, parent_id, author_id, tags, action_type}) => {
-  let query = CommentModel.find();
+const getCommentCountByQuery = (ctx, options) => {
+  const {statuses, asset_id, parent_id, author_id, tags, action_type} = options;
 
   // If user queries for statuses other than NONE and/or ACCEPTED statuses, it needs
   // special privileges.
   if (
     (!statuses || statuses.some((status) => !['NONE', 'ACCEPTED'].includes(status))) &&
-    (context.user == null || !context.user.can(SEARCH_NON_NULL_OR_ACCEPTED_COMMENTS))
+    (ctx.user == null || !ctx.user.can(SEARCH_NON_NULL_OR_ACCEPTED_COMMENTS))
   ) {
     return null;
   }
 
-  if (statuses && statuses.length > 0) {
-    query = query.where({status: {$in: statuses}});
-  }
-
-  if (ids) {
```

Comment:
This is not required anymore?

Reply:
We don't actually use that query anymore no. That was used before when we had to search for all the flags and then find all the comments from those flags with the given id's. The query paths in place now never use the `ids`.